js/viewer/annotation/base.js

```diff
@@ -215,6 +214,10 @@ Base.prototype.renderSVG = function(){
 
     // add all the attributes
     for(attr in this._attrs){
+        // We skip the marker-end attribute if the annotation is not arrow line
```

Comment:
Why do you need to add this check? Didn't you add these attributes only for the arrowline?

Reply:
Removed the check, previously the marker-end attribute was in base.js as well. I thought that it needed to be present in the allowed attributes to be added to the arrow line and it was getting added to other annotations too. Now, that I’ve removed the attribute from base.js, the check is no longer needed.